frontend/packages/console-shared/src/selectors/node.ts

```diff
@@ -66,3 +66,8 @@ export const isWindowsNode = (node) =>
 
 export const getNodeUptime = (node: NodeKind) =>
   _.find(node?.status?.conditions, { type: 'Ready', status: 'True' })?.lastTransitionTime;
+
+export const getCpuArch = (node: NodeKind): string => {
+  const conditions = _.get(node, 'status.nodeInfo');
+  return conditions.architecture;
+};
```

Comment:
```suggestion
```

Reply:
Thanks for the suggestions , removed this code .